src/modules/price-router/Extensions/EtherFi/eETHExtension.sol

```diff
@@ -0,0 +1,60 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.21;
+
+import { Extension, PriceRouter, ERC20, Math } from "src/modules/price-router/Extensions/Extension.sol";
+import { IRateProvider } from "src/interfaces/external/EtherFi/IRateProvider.sol";
+
+/**
+ * @title Sommelier Price Router eETH Extension
+ * @notice Allows the Price Router to price eETH.
+ * @author 0xEinCodes
+ */
+contract eEthExtension is Extension {
+    using Math for uint256;
+
+    constructor(PriceRouter _priceRouter) Extension(_priceRouter) {}
+
+    /**
+     * @notice Attempted to add eETH support when weETH is not supported.
+     */
+    error eEthExtension__WEETH_NOT_SUPPORTED();
+
+    /**
+     * @notice Attempted to use this extension to price something other than eETH.
+     */
+    error eEthExtension__ASSET_NOT_EETH();
+
+    /**
+     * @notice Ethereum mainnet eETH.
+     */
+    address public eETH = 0x35fA164735182de50811E8e2E824cFb9B6118ac2;
```

Comment:
Make these constants or immutable

Reply:
lol why did I not make the wsteth ones constant

Reply:
Also make these ERC20s

Reply:
Changes have been made :)